### PR TITLE
DBZ-8063 Add documentation for webhook auth config

### PR DESCRIPTION
**Pull request by TimoWilhelm**
Update documentation of config options for Standard Webhooks authentication.

Main PR: https://github.com/debezium/debezium-server/pull/108

### CODE REVIEW

**Review by Naros — COMMENTED** [review]
Overall LGTM, just asked Bob for a quick cursory review.

**Review by roldanbob — CHANGES_REQUESTED** [review]
A few small suggestions.
Also, please adhere to the standard of entering each sentence on its own line.
Doing so makes the doc sources easier to read and it also make it easier to manage diffs.
It's true that this was not always done, but it's the preferred format for all newly added content.
Thanks!

**Comment by TimoWilhelm** [comment]
Thanks for the review! I updated the docs with the suggestions.

File: documentation/modules/ROOT/pages/operations/debezium-server.adoc [hunk 1]
```diff
@@ -798,7 +798,13 @@ optional https://en.wikipedia.org/wiki/JSON_Web_Token[JSON Web Token (JWT) authe
...
 |[[httpclient-authentication-type]]<<httpclient-authentication-type, `debezium.sink.http.authentication.type` >>
 |
-|Specifies the authentication type to use.  If missing, no authentication is used.  Currently, only JSON Web Token (JWT) authentication (indicated by the value `jwt`) is supported.
+|Specifies the type of authentication the HTTP client sink uses when connecting to an HTTP server.  
+Supports one of the following options:
+
+`jwt`:: JSON Web Token (JWT) authentication.
+`standard-webhooks`:: link:https://www.standardwebhooks.com/[Standard Webhooks].
+
+If you omit this property, the HTTP client sink will not use any authentication headers for the connection.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/operations/debezium-server.adoc:807]:
```suggestion
If you omit this property, the HTTP client sink does not use authentication headers for the connection.
```

File: documentation/modules/ROOT/pages/operations/debezium-server.adoc [hunk 2]
```diff
@@ -820,6 +826,13 @@ optional https://en.wikipedia.org/wiki/JSON_Web_Token[JSON Web Token (JWT) authe
 |
 |Requested duration (in minutes) before the refresh token expires.
 
+|[[httpclient-authentication-webhook-secret]]<<httpclient-authentication-webhook-secret, `debezium.sink.http.webhook.secret` >>
+|
+|The webhooks signing secret that {prodname} uses to generate HMAC-SHA256 signatures of the webhook request.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/operations/debezium-server.adoc:831]:
```suggestion
|The webhooks signing secret that {prodname} uses to generate HMAC-SHA256 signatures for the webhook request. 
```

File: documentation/modules/ROOT/pages/operations/debezium-server.adoc [hunk 3]
```diff
@@ -820,6 +826,13 @@ optional https://en.wikipedia.org/wiki/JSON_Web_Token[JSON Web Token (JWT) authe
 |
 |Requested duration (in minutes) before the refresh token expires.
 
+|[[httpclient-authentication-webhook-secret]]<<httpclient-authentication-webhook-secret, `debezium.sink.http.webhook.secret` >>
+|
+|The webhooks signing secret that {prodname} uses to generate HMAC-SHA256 signatures of the webhook request. 
+The secret must be Base64-encoded, with a size from 24 bytes to 64 bytes (192–512 bits).
+Optionally, you can add the prefix `whsec_` to the secret to help distinguish it from other types of keys or tokens.
+For more information about implementing or validating Webhook signatures, see the link:https://github.com/standard-webhooks/standard-webhooks/blob/main/spec/standard-webhooks.md[Standard Webhooks specification].
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/operations/debezium-server.adoc:834]:
```suggestion
For more information about implementing or validating webhook signatures, see the link:https://github.com/standard-webhooks/standard-webhooks/blob/main/spec/standard-webhooks.md[Standard Webhooks specification]. 
```
In general, when "webhooks" is not part of a title or heading, and does not refer to a particular company or product, the lower case is the accepted form.

File: documentation/modules/ROOT/pages/operations/debezium-server.adoc (hunk 1 above)
Comment (roldanbob) [reply documentation/modules/ROOT/pages/operations/debezium-server.adoc:807]: Write in the simple present tense as much a possible.

**Review by roldanbob — CHANGES_REQUESTED** [review]
Hi @TimoWilhelm. Thanks for your quick response. I noticed a couple of further instances where we need minor changes.

**Comment by TimoWilhelm** [comment]
@roldanbob Thanks, I applied the suggested changes.

**Review by roldanbob — APPROVED** [review]
LGTM, @TimoWilhelm Thanks for your efforts and patience!

File: documentation/modules/ROOT/pages/operations/debezium-server.adoc [hunk 4]
```diff
@@ -820,6 +826,13 @@ optional https://en.wikipedia.org/wiki/JSON_Web_Token[JSON Web Token (JWT) authe
 |
 |Requested duration (in minutes) before the refresh token expires.
 
+|[[httpclient-authentication-webhook-secret]]<<httpclient-authentication-webhook-secret, `debezium.sink.http.webhook.secret` >>
```
Comment (mfvitale) [inline documentation/modules/ROOT/pages/operations/debezium-server.adoc:829]: @TimoWilhelm  Looking at the code this seems should be `debezium.sink.http.authentication.webhook.secret`. Ins't it?

**Review by mfvitale — APPROVED** [review]

**Pull request merged**